web/layouts/partials/header.html

```diff
@@ -18,6 +18,7 @@
           <li class="dropdown">
             <a role="button" aria-haspopup="true" aria-expanded="false">Contribute</a>
             <div class="dropdown-content">
+              <a href="https://www.theforeman.org">Project site</a>
               <a href="https://community.theforeman.org">Discourse forum</a>
```

Comment:
I stuffed these two changes as well - clicking on logo gets you back to the welcome page and there's new Project site link in the Contribute menu now. This feels better to me.